test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -217,6 +217,17 @@ Describe "Get-ChildItem" -Tags "CI" {
         It "Should list the folder present when path length equal to MAX_PATH" {
             (Get-ChildItem -Path TestDrive:\$item_I -Recurse -Force).Name.Length | Should -BeGreaterThan 0
         }
+
+        It 'Trailing slash for -Path should treat it as a folder only when used with -recurse' {
+            $foo = New-Item -ItemType Directory -Path TestDrive:\foo
+            $foo2 = New-Item -ItemType Directory -Path TestDrive:\foo\foo
+            $bar = New-Item -ItemType File -Path TestDrive:\foo\bar
+            $bar2 = New-Item -ItemType File -Path TestDrive:\foo\foo\bar
```

Comment:
The tests work as-is but if I tried replace `bar` with another name (ex. `dar`) the test doesn't work!

Reply:
Just to be clear, you replaced all the instances of `bar` with `dar` in the test and the test fails?

Reply:
I downloaded a build from the PR and it works as expected for my case too. So I close my question.